docker/files/elasticsearch.yml

```diff
@@ -10,4 +10,10 @@ http.cors.allow-origin: "*"
 http.cors.enabled: true
 index.max_result_window: 1000000
 
-action.auto_create_index: false
+# Eventually we want to enable this option, but currently it causes ES
+# timeouts due to tests referencing non-existent indices. For example,
+# saving a user triggers an async task that updates ES, and that fails
```

Comment:
Hey @millerdev  just catching up on this issue. So we need to do this sooner than latter as some of ES work is dependent on this such as [this one](https://github.com/dimagi/commcare-hq/pull/26733) and upcoming ES 7 upgrade.

Ideally such test cases where they expect a non-existent index should fail, so that we can fix and not timeout like this. I am surprised this didn't happen when I put a PR to enable this.

Can you give the example tests that timed out, so I can try to re-add this setting?

Reply:
@sravfeyn I'm getting close to merging https://github.com/dimagi/commcare-hq/pull/27433, which should make it easier to tell which tests are affected by this option. If you want to test it out before I get that merged, feel free to make a new branch and draft PR off of [dm/slow-test-fail](https://github.com/dimagi/commcare-hq/tree/dm/slow-test-fail) and re-enable this option on your new branch.

You can also see the effect it had by looking at [average test times in datadog](https://app.datadoghq.com/dashboard/jmh-4qt-sp8/travis-test-timings?from_ts=1586963957824&fullscreen_end_ts=1589555983879&fullscreen_paused=true&fullscreen_section=overview&fullscreen_start_ts=1586963983879&fullscreen_widget=184804293&live=true&to_ts=1589555957824). Note how average test times on node 4 (`partition:,environment:travis`) nearly doubled between 2020/04/29 and 2020/05/08.

![image](https://user-images.githubusercontent.com/464726/82067646-ffde9d80-969e-11ea-9a27-c39133dd6284.png)

Reply:
Thanks Daniel, I put out a PR now to see how tests go.